diff --git a/node_modules/@microsoft/fetch-event-source/package.json b/node_modules/@microsoft/fetch-event-source/package.json
index 8528735..2e9bac3 100644
--- a/node_modules/@microsoft/fetch-event-source/package.json
+++ b/node_modules/@microsoft/fetch-event-source/package.json
@@ -9,9 +9,9 @@
   },
   "author": "Microsoft",
   "license": "MIT",
-  "main": "lib/cjs/index.js",
-  "module": "lib/esm/index.js",
-  "types": "lib/cjs/index.d.ts",
+  "main": "src/index.ts",
+  "module": "src/index.ts",
+  "types": "src/index.ts",
   "sideEffects": false,
   "scripts": {
     "clean": "rimraf ./lib ./coverage",
diff --git a/node_modules/@microsoft/fetch-event-source/src/fetch.ts b/node_modules/@microsoft/fetch-event-source/src/fetch.ts
index 1e2236c..ccfdf6d 100644
--- a/node_modules/@microsoft/fetch-event-source/src/fetch.ts
+++ b/node_modules/@microsoft/fetch-event-source/src/fetch.ts
@@ -84,7 +84,7 @@ export function fetchEventSource(input: RequestInfo, {
         }
 
         let retryInterval = DefaultRetryInterval;
-        let retryTimer = 0;
+        let retryTimer : ReturnType<typeof globalThis['setTimeout']> | undefined = undefined;
         function dispose() {
             globalThis.document?.removeEventListener('visibilitychange', onVisibilityChange);
             globalThis.clearTimeout(retryTimer);
diff --git a/node_modules/@microsoft/fetch-event-source/src/index.ts b/node_modules/@microsoft/fetch-event-source/src/index.ts
index cdcc974..f793fb1 100644
--- a/node_modules/@microsoft/fetch-event-source/src/index.ts
+++ b/node_modules/@microsoft/fetch-event-source/src/index.ts
@@ -1,2 +1,3 @@
-export { fetchEventSource, FetchEventSourceInit, EventStreamContentType } from './fetch';
-export { EventSourceMessage } from './parse';
+export { fetchEventSource, type EventStreamContentType, type FetchEventSourceInit } from './fetch';
+export { type EventSourceMessage } from './parse';
+
diff --git a/node_modules/@microsoft/fetch-event-source/src/parse.ts b/node_modules/@microsoft/fetch-event-source/src/parse.ts
index c13e3c4..40af9aa 100644
--- a/node_modules/@microsoft/fetch-event-source/src/parse.ts
+++ b/node_modules/@microsoft/fetch-event-source/src/parse.ts
@@ -1,3 +1,5 @@
+
+
 /**
  * Represents a message sent in an event stream
  * https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events/Using_server-sent_events#Event_stream_format
@@ -21,7 +23,7 @@ export interface EventSourceMessage {
  */
 export async function getBytes(stream: ReadableStream<Uint8Array>, onChunk: (arr: Uint8Array) => void) {
     const reader = stream.getReader();
-    let result: ReadableStreamDefaultReadResult<Uint8Array>;
+    let result: ReadableStreamReadResult<Uint8Array>;
     while (!(result = await reader.read()).done) {
         onChunk(result.value);
     }
