diff --git a/node_modules/pocketbase/dist/pocketbase.es.d.ts b/node_modules/pocketbase/dist/pocketbase.es.d.ts
index cdc523e..8df8503 100644
--- a/node_modules/pocketbase/dist/pocketbase.es.d.ts
+++ b/node_modules/pocketbase/dist/pocketbase.es.d.ts
@@ -405,6 +405,7 @@ interface AuthMethodsList {
     emailPassword: boolean;
     authProviders: Array<AuthProviderInfo>;
 }
+
 interface RecordSubscription<T = Record> {
     [key: string]: any;
     action: string;
@@ -847,4 +848,4 @@ declare class LocalAuthStore extends BaseAuthStore {
 declare function getTokenPayload(token: string): {
     [key: string]: any;
 };
-export { Client as default, ClientResponseError, BaseAuthStore, LocalAuthStore, getTokenPayload, ExternalAuth, Admin, Collection, Record, LogRequest, BaseModel, ListResult, SchemaField };
+export { Client as default, ClientResponseError, BaseAuthStore, LocalAuthStore, getTokenPayload, ExternalAuth, Admin, Collection, Record, LogRequest, BaseModel, ListResult, SchemaField, RecordSubscription };
