diff --git a/node_modules/pocketbase/dist/pocketbase.es.d.ts b/node_modules/pocketbase/dist/pocketbase.es.d.ts
index 1146925..d1265e8 100644
--- a/node_modules/pocketbase/dist/pocketbase.es.d.ts
+++ b/node_modules/pocketbase/dist/pocketbase.es.d.ts
@@ -380,7 +380,7 @@ declare class RecordService extends CrudService<Record> {
     /**
      * Subscribe to realtime changes of any record from the collection.
      */
-    subscribe<T = Record>(callback: (data: RecordSubscription<T>) => void): Promise<void>;
+    subscribe<T = Record>(topic: string, callback: (data: RecordSubscription<T>) => void): Promise<()=>void>;
     /**
      * Subscribe to the realtime changes of a single record in the collection.
      */
@@ -812,4 +812,4 @@ declare class LocalAuthStore extends BaseAuthStore {
 declare function getTokenPayload(token: string): {
     [key: string]: any;
 };
-export { Client as default, ClientResponseError, BaseAuthStore, LocalAuthStore, getTokenPayload, ExternalAuth, Admin, Collection, Record, LogRequest, BaseModel, ListResult, SchemaField };
+export { Client as default, ClientResponseError, BaseAuthStore, LocalAuthStore, getTokenPayload, ExternalAuth, Admin, Collection, Record, LogRequest, BaseModel, ListResult, SchemaField, RecordSubscription };
