diff --git a/node_modules/sqlite3/lib/sqlite3.d.ts b/node_modules/sqlite3/lib/sqlite3.d.ts
index b27b0cf..1921b6c 100644
--- a/node_modules/sqlite3/lib/sqlite3.d.ts
+++ b/node_modules/sqlite3/lib/sqlite3.d.ts
@@ -139,6 +139,19 @@ export class Database extends events.EventEmitter {
     wait(callback?: (param: null) => void): this;
 
     interrupt(): void;
+
+    backup(path:string): Backup
+
+}
+
+export class Backup extends events.EventEmitter {
+    idle: boolean
+    completed: boolean
+    failed: boolean
+    remaining: number
+    pageCount: number
+    finish():void
+    step(nPages: number): void
 }
     
 export function verbose(): sqlite3;
